.github/workflows/push-docs.yaml

```diff
@@ -164,52 +150,52 @@ jobs:
         sed -i '1,2d' assets/conrefs/pages/gateway/reference/gateway_parameters_${{ steps.version-variables.outputs.minor }}.md
         popd || exit 1
     - name: Push and create PR
+      uses: peter-evans/create-pull-request@v7
```

Comment:
Example of a successful run from the workflow file in this branch: https://github.com/solo-io/gloo/actions/runs/14541347331/job/40799827433